bin/commands/install/index.ts

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
This code is not executed yet. `zwe install` works as `shell` only.

User prepares some `zowe.yaml` file and verify `zowe.setup.dataset.prefix` is there:
```
$: zwe config get -c zowe.yaml -p '.zowe.setup.dataset.prefix'
Temporary directory '/tmp/.zweenv-8536' created.
Zowe will remove it on success, but if zwe exits with a non-zero code manual cleanup would be needed.
IBMUSER.ZWEV2
```
`zowe.setup.dataset.prefix` is defined, because it is taken from `defaults.yaml`. Now `$: zwe install -c zowe.yaml` returns:
```
===============================================================================
>> INSTALL ZOWE MVS DATA SETS

Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined in Zowe YAML configuration file.
```
Merged config is not accessible by `zwe install`.

Reply:
I am pulling this from the PR for now.
It just wasn't in scope so I didn't want to make testing more complicated.

Reply:
This problem should be solved by removing `zowe.setup.dataset.prefix` from `defaults.yaml`. Otherwise the code is not executed, it can stay (or not).

Reply:
Done that too.